webapp/src/shared/components/Row/Row.tsx

```diff
@@ -3,10 +3,12 @@ import './Row.css';
 import { TextColor, TextVariant, TextWeight } from '../Text/Text';
 import { default as Text } from '../Text/Text';
 import { AvatarProps, SmallAvatar } from '../Avatar/Avatar';
+import { Link } from 'react-router-dom';
 
 export type RowProps = {
   iconVariant?: IconVariant;
   onClick?: () => void;
+  url?: string;
```

Comment:
suggestion: Here we could make this component require one of the `url` or `onClick` and disallow both or none.
https://stackoverflow.com/a/61281828

Reply:
Does this solution work for you?